<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>833</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BW_WINCC</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>BenewakeLogo.ico</normaloff>BenewakeLogo.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>451</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton_connect">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>8</y>
       <width>120</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>连接串口</string>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_serial">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>15</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>串口</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_serial">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>12</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_baudrate">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>50</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>波特率</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_baudrate">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>47</y>
       <width>120</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>256000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>460800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>921600</string>
      </property>
     </item>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>65</y>
       <width>441</width>
       <height>41</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_refresh">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>45</y>
       <width>120</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>刷新串口</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget1" native="true">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>89</y>
      <width>451</width>
      <height>711</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>452</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>语言(Language)</string>
    </property>
    <addaction name="actionChinese"/>
    <addaction name="separator"/>
    <addaction name="actionEnglish"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionChinese">
   <property name="text">
    <string>中文</string>
   </property>
   <property name="font">
    <font>
     <underline>false</underline>
     <kerning>false</kerning>
    </font>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>trigger_actOpen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>330</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_connect</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>connectSerial()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>426</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_refresh</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>refreshSerial()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>536</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>431</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>trigger_actOpen()</slot>
  <slot>getSerialPort()</slot>
  <slot>connectSerial()</slot>
  <slot>refreshSerial()</slot>
 </slots>
</ui>
